util/include/util/filesystem.hpp

```diff
@@ -0,0 +1,22 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#if __has_include(<filesystem>)
+#include <filesystem>
+namespace fs = std::filesystem;
```

Comment:
Two questions:
1. Do we use a compiler on which the filesystem library is still experimental?
2. Shouldn't the specific files who want this alias do it? A global two character namespace seems risky to me.

Reply:
1. it doesn't hurt
2. although it's a common namespace alias, we can replace it by a longer one.

Reply:
1. There might be incompatibilities between experimental and final version, which will result is difficult bugs, so If this is not necessary, I wouldn't do it.
2. bft::fs is probably best if we are using our own header.